docs/content/documentation/systems/Controls/index.md

```diff
@@ -1,7 +1,81 @@
-<!-- MOOSE Documentation Stub: Remove this when content is added. -->
-
 # Controls System
+
+The control system in MOOSE has one primary purpose: ** to modify input parameters during runtime
+of a MOOSE-based simulation.**
+
+## Creating a Controllable Parameter
+
+The input parameters of objects you which to be controlled must:
+
+### Store Parameter as a `const` Reference
+In your *.h files, declare storage for the parameters as follows.
+
+!listing framework/include/bcs/DirichletBC.h line=_value
+
+In the *.C file initialize the reference in the constructor initialization lists as:
+
+!listing framework/src/bcs/DirichletBC.C line=_value(getParam
+
+### Declare the Parameter as Controllable
+
+In order to "control" a parameter it must be communicated that the parameter is allowed to be
+controlled, this is done in the `validParams` function, as shown below.
+
+
+!listing framework/src/bcs/DirichletBC.C start=template end=DirichletBC::
+
+!!!note
+    The `declareControllable` method also accepts a space separated list of parameters.
+
+
+## Create a Control object
+
+`Control` objects are similar to other systems in MOOSE. You create a control in your application
+by inheriting from the `Control` C++ class in MOOSE. It is required to override the `execute`
+method in your custom object. within this method the following methods are generally used to get
+or set controllable parameters:
+
+  * `getControllableValue` <br>
+  This method returns the current controllable parameter, in the case that multiple parameters are
+  being controlled, only the first value will be returned and a warning will be produced if the
+  values are differ (this warning may be disabled).
+
+  * `setControllableValue` <br>
+  This will method allows for a controllable parameter to be changed, in the case that multiple
+  parameters are being controlled, all of the values will be set.
+
+There are additional methods as well is overloaded versions of these methods that allow for the
```

Comment:
"methods with(?) overloaded versions of these methods" - "is" there does not make sense to me.

Maybe you wanted to say: "There are additional overloaded methods that allow...". The sentence does not read well...